repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/samlpolicy/SamlFlow10Test.groovy

```diff
@@ -128,7 +130,16 @@ class SamlFlow10Test extends ReposeValveTest {
 
     @Unroll
     def "a saml:response that #signedState will not be altered and will maintain signature validity through Repose"() {
-        when:
+        given: "the Identity mock will return a mapping policy with extended attributes should one be requested (it shouldn't be)"
```

Comment:
If it shouldn't be requested wouldn't we be better off not setting this behavior to prove our assumptions?